app/src/main/res/layout/movie_list_fragment.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
@ifeins that was the default layout created, can you point me to resources about the performance thing? I searched (quickly) and didn't find anything